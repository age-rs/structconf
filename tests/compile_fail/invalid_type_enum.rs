use structconf::StructConf;

#[derive(StructConf)]
struct Enum {
    A,
    B,
}

fn main() {}
