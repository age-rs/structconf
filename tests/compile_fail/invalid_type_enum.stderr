error: expected `:`, found `,`
 --> $DIR/invalid_type_enum.rs:5:6
  |
5 |     A,
  |      ^ expected `:`
